i18n/en.i18n.json

```diff
@@ -185,6 +185,8 @@
   "LDAP_Dn" : "Distinguished Name (DN)",
   "LDAP_Enable" : "Enable LDAP",
   "LDAP_Port" : "LDAP Port",
+  "LDAP_Sync_User_Data" : "Keep user data in sync with server",
+  "LDAP_Sync_User_Data_FieldMap" : "User data field map",
```

Comment:
@gmsecrieru I submitted [!1310](https://github.com/RocketChat/Rocket.Chat/pull/1310) which should alleviate some of my concerns in this thread.